lib/src/components/NxTransferList/TransferListHalf.scss

```diff
@@ -85,23 +85,32 @@ $-visible-item-count: 10;
     #{$-visible-item-count - 1} * #{$-item-vertical-margin}
   );
 
-
   overflow-y: auto;
   padding: var(--nx-spacing-6x) $-control-box-side-spacing;
   margin-top: var(--nx-spacing-6x);
 }
 
 .nx-transfer-list__item {
   @include nx-container-helpers.container-horizontal;
-  @include nx-text-helpers.truncate-ellipsis;
 
   background-color: var(--nx-swatch-white);
-  border: var(--nx-border-default);
   border-radius: var(--nx-border-radius);
+  display: flex;
+  justify-content: space-between;
+  margin: var(--nx-spacing-2x) 0;
+  padding: 0;
+}
+
+.nx-transfer-list__select {
+  @include nx-container-helpers.container-horizontal;
+  @include nx-text-helpers.truncate-ellipsis;
+
+  border-radius: var(--nx-border-radius);
+  border: var(--nx-border-default);
   cursor: pointer;
   display: block;
```

Comment:
Not needed

Reply:
> Not needed

Removed: 8da7e29ba73263e6ce9f96e12d36ba9915305d0f